tools/base/src/Base.ts

```diff
@@ -10,6 +10,7 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
+import { Theme } from '@spectrum-web-components/theme';
```

Comment:
This breaks everything. I'm not sure what I'm doing wrong, but I get this error when running `yarn install`

```
tools/base/src/Base.ts:13:23 - error TS2307: Cannot find module '@spectrum-web-components/theme' or its corresponding type declarations.

13 import { Theme } from '@spectrum-web-components/theme';
                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
(it was `tsconfig.json`'s `references` array)